.tekton/run_unittests.sh

```diff
@@ -29,21 +29,21 @@ default)
   esac
   export TESTS=('tests') ;;
 cassandra)
-  export REQUIREMENTS='requirements-cassandra.txt'
-  export TESTS=('tests/clients/test_cassandra-driver.py')
-  export CASSANDRA_TEST='true' ;;
-couchbase)
-  export REQUIREMENTS='requirements-couchbase.txt'
-  export TESTS=('tests/clients/test_couchbase.py')
-  export COUCHBASE_TEST='true' ;;
-gevent_starlette)
-  export REQUIREMENTS='requirements-gevent-starlette.txt'
-  export TESTS=('tests/frameworks/test_gevent.py' 'tests/frameworks/test_starlette.py')
-  export GEVENT_STARLETTE_TEST='true' ;;
+#   export REQUIREMENTS='requirements-cassandra.txt'
+#   export TESTS=('tests/clients/test_cassandra-driver.py')
+#   export CASSANDRA_TEST='true' ;;
+# couchbase)
+#   export REQUIREMENTS='requirements-couchbase.txt'
+#   export TESTS=('tests/clients/test_couchbase.py')
+#   export COUCHBASE_TEST='true' ;;
+# gevent_starlette)
+#   export REQUIREMENTS='requirements-gevent-starlette.txt'
+#   export TESTS=('tests/frameworks/test_gevent.py' 'tests/frameworks/test_starlette.py')
+#   export GEVENT_STARLETTE_TEST='true' ;;
 *)
-  echo "ERROR \$TEST_CONFIGURATION='${TEST_CONFIGURATION}' is unsupported " \
-       "not in (default|cassandra|couchbase|gevent_starlette)" >&2
-  exit 3 ;;
+  # echo "ERROR \$TEST_CONFIGURATION='${TEST_CONFIGURATION}' is unsupported " \
+  #      "not in (default|cassandra|couchbase|gevent_starlette)" >&2
+  # exit 3 ;;
```

Comment:
Same as [this](https://github.com/instana/python-sensor/pull/571/files#r1698713275)

Reply:
Same as [https://github.com/instana/python-sensor/pull/571#discussion_r1699090486](this)

Reply:
> Same as [https://github.com/instana/python-sensor/pull/571#discussion_r1699090486](this)

In that case, can you uncomment the lines in this file and see if it works? I think since you've commented out in the other 2 files, this wouldn't be called at all.

Reply:
I've reverted everything related to tekton, now.